api/tests/src/data_migration/test_setup_foreign_tables.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from src.data_migration.setup_foreign_tables import Column, Constants, build_sql
+
+EXPECTED_LOCAL_OPPORTUNITY_SQL = "CREATE TABLE IF NOT EXISTS foreign_topportunity (OPPORTUNITY_ID numeric(20) CONSTRAINT TOPPORTUNITY_pkey PRIMARY KEY NOT NULL,OPPNUMBER character varying (40),REVISION_NUMBER numeric(20),OPPTITLE character varying (255),OWNINGAGENCY character varying (255),PUBLISHERUID character varying (255),LISTED CHAR(1),OPPCATEGORY CHAR(1),INITIAL_OPPORTUNITY_ID numeric(20),MODIFIED_COMMENTS character varying (2000),CREATED_DATE DATE,LAST_UPD_DATE DATE,CREATOR_ID character varying (50),LAST_UPD_ID character varying (50),FLAG_2006 CHAR(1),CATEGORY_EXPLANATION character varying (255),PUBLISHER_PROFILE_ID numeric(20),IS_DRAFT character varying (1))"
```

Comment:
Should be possible to wrap this onto multiple lines using implicit string joining:

```suggestion
EXPECTED_LOCAL_OPPORTUNITY_SQL = (
    "CREATE TABLE IF NOT EXISTS foreign_topportunity "
    "(OPPORTUNITY_ID numeric(20) CONSTRAINT TOPPORTUNITY_pkey PRIMARY KEY NOT NULL,"
    "OPPNUMBER character varying (40),"
    "REVISION_NUMBER numeric(20),"
    "OPPTITLE character varying (255),"
    "OWNINGAGENCY character varying (255),"
    "PUBLISHERUID character varying (255),"
    "LISTED CHAR(1),"
    "OPPCATEGORY CHAR(1),"
    "INITIAL_OPPORTUNITY_ID numeric(20),"
    "MODIFIED_COMMENTS character varying (2000),"
    "CREATED_DATE DATE,"
    "LAST_UPD_DATE DATE,"
    "CREATOR_ID character varying (50),"
    "LAST_UPD_ID character varying (50),"
    "FLAG_2006 CHAR(1),"
    "CATEGORY_EXPLANATION character varying (255),"
    "PUBLISHER_PROFILE_ID numeric(20),"
    "IS_DRAFT character varying (1))"
)
```